reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-73.md

```diff
@@ -129,16 +122,20 @@ For more information about the Experimental Features, see [Using Experimental Fe
     **System.Security.AccessControl.ObjectSecurity**.
   - After this change, a user needs to explicitly import **Microsoft.PowerShell.Security** module in
     order to use those code properties and code methods.
+- In this release, Windows APIs were updated or removed for compliance, which means that PowerShell
+  7.3 doesn't run on Windows 7. While Windows 7 is no longer supported, previous builds could run on
+  Windows 7.
```

Comment:
I think we might want to move this further up the list, maybe to the top, as it's dropping support for an entire OS version.